internal/pkg/otel/samples/darwin/logs_metrics_traces.yml

```diff
@@ -89,14 +322,52 @@ processors:
       - key: process.executable.path
         action: delete
 
-exporters:
-  otlp/apm:
-    endpoint: "${env:APM_ENDPOINT}"
-    headers:
-    # Elastic APM Server secret token or API key
-      Authorization: "Bearer ${env:APM_SECRET_KEY}"
+  elastictrace:
+  lsminterval:
```

Comment:
Unrelated to this PR but pointing it out here: we should also look into mounting a volume for lsminterval to use disk instead of memory for aggregation. We could probably start with `emptyDir` but maybe worth looking into mounting a presistent volume too.